package.json

```diff
@@ -2,15 +2,27 @@
   "name": "@waku/react",
```

Comment:
Do you have permissions to publish?
Also, repo is `waku-ui`. Is the idea to make it a monorepo when we decided to add more component for other ui frameworks?

Reply:
Yes, idea behind is to have more frameworks supported in the future. For now only React is there.